.gulp/regeneration.iced

```diff
@@ -100,7 +100,10 @@ rubyAzureMappings = {
 
 rubyAzureAdditionalMappings = {
   'azure_resource_inheritance': ['resource_inheritance.json', 'AzureResourceInheritanceModule'],
-  'odata_type_property': ['odata_type_property.json', 'AzureODataTypePropertyModule'],
+  'odata_type_property': ['odata_type_property.json', 'AzureODataTypePropertyModule']
+}
+
+rubyAzureValidationsMappings = {
   'parameters_constraints': ['parameters_constraints.json', 'AzureParametersConstraintsModule']
```

Comment:
could we just have this as part of the rubyMappings ?
same with 'constrains_no_validation' ?

Reply:
No. Ruby mappings looks for the files within autorest.